src/types/params.ts

```diff
@@ -0,0 +1,17 @@
+// Common params across multiple phases, according to
+// https://github.com/babylonlabs-io/pm/blob/main/adr/adr-023-update-btcstaking-params.md
```

Comment:
`pm` is a private repository. Further, these are the params of the btcstaking module for the Babylon blockchain, they do not necessarily coincide with the whole picture.

Reply:
Note that according to this document, we also need `slashing_rate` and `min_slashing_fee` (among others, didn't check in depth). Also the unbonding time is a minimum, not a static value. We can explore how to make these params phase-2 compliant in a future PR (and track it as an issue for now).